404-server/Makefile

```diff
@@ -27,12 +27,12 @@ GOLANG_VERSION=1.7
 TEMP_DIR:=$(shell mktemp -d)
 
 server: server.go
-	CGO_ENABLED=0 GOOS=linux GOARCH=$(ARCH) GOARM=6 go build -a -installsuffix cgo -ldflags '-w -s' -o server ./server.go
+	CGO_ENABLED=0 GOOS=linux GOARCH=$(ARCH) GOARM=6 go build -a -installsuffix cgo -ldflags '-w -s' -o server ./server.go ./metrics.go
```

Comment:
nit: I guess `.` or nothing just works here as well?
Not optimal to enumerate all files here if/as the source grows in size